src/components/Header/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, useState } from "react";
+
+import Navbar from "@components/Navbar";
+import Logo from "@images/Logo.svg";
+import { log } from "@utils/index";
+import { AiOutlineClose } from "react-icons/ai";
+import { BiUser } from "react-icons/bi";
+import { BsHandbag } from "react-icons/bs";
+import { GiHamburgerMenu } from "react-icons/gi";
+
+import styles from "./Header.module.scss";
+
+type HeaderProps = {};
+
+const Header: FC<HeaderProps> = () => {
+  const [visibleMenu, setVisibleMenu] = useState<boolean>(false);
+  const burgerMenu = () => {
```

Comment:
если это функция а не значение, то и называй её соответствующе
```suggestion
  const toggleBurgerMenu = () => {
```
и стоит обернуть в `useCallback`